### PR TITLE
[DOC] add alexfilothodoros to contributors

### PR DESCRIPTION
**Pull request by alexfilothodoros**
Adds @alexfilothodoros as a contributor for #5169

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -2330,3 +2330,14 @@
     }
   ]
 }
```
Comment (fkiraly) [inline .all-contributorsrc:2332]: please ensure you follow the pattern of brackets & commas

**Review by fkiraly — CHANGES_REQUESTED** [review]
formatting issue, see above.

Otherwise fine once https://github.com/sktime/sktime/pull/5169 is merged.

Also, I think the badge should be "maintenance", not "example", since you didn't add content, but changed content so the tests and notebooks run faster.

**Review by fkiraly — APPROVED** [review]
fixed the formatting to ensure this is included in the automatic contributor update

**Comment by alexfilothodoros** [comment]
> fixed the formatting to ensure this is included in the automatic contributor update

Thanks for that :)

**Pull request merged**